server/utils/referralUtils.test.ts

```diff
@@ -91,7 +91,11 @@ describe('ReferralUtils', () => {
               url: `/referrals/${referral.id}/oasys-confirmed`,
             },
             {
-              statusTag: { classes: 'govuk-tag--grey moj-task-list__task-completed', text: 'not started' },
+              statusTag: {
+                attributes: { 'data-testid': 'referral-reason-tag' },
```

Comment:
Everything is `referral` so I think this can just be `reason-tag`? There shouldn't be a conflict on the task list page